Utilities/ScanSubscription.ps1

```diff
@@ -0,0 +1,390 @@
+# Description: This script is used for checking enabled VM sizes and VM core quota in the given azure subscription.
+# It will upload subscription id and unavailable VM/insufficient VM core to db
+# This script will be executed by 'Penguinator-Validate-Azure-Subscription-Info' pipeline
+# This script should be under .\lisav2\Utilities directory
+
+Param
+(
+    [String] $clientSecret,
+    [String] $clientID,
+    [String] $tenantID,
+    [String] $subID,
+    [String] $lisaTestCasesXMLPath = '.\lisav2\XML\TestCases',
```

Comment:
"lisav2" seems to be a custom folder, can we set it to be ".\xml\TestCases", and assume the current execution folder is the cloned repo folder?

Reply:
Currently (I mean in my test pipeline) I clone lisa to $executionFolder/lisav2 to differentiate lisa codes from other codes/files under execution folder. I prefer keep it as is.

Reply:
you can add "pushd lisa" in your pipeline job.
If we want to keep it as a generic utility, it's better not having the customized path, or we can pass "lisa" from parameter